bodo/libs/_distributed.h

```diff
@@ -993,21 +993,24 @@ static void c_gather_scalar(void* send_data, void* recv_data, int typ_enum,
     return;
 }
 
-static void c_gatherv(void* send_data, int sendcount, void* recv_data,
-                      int* recv_counts, int* displs, int typ_enum,
+static_assert(sizeof(MPI_Count) == sizeof(int64_t));
+static_assert(sizeof(MPI_Aint) == sizeof(int64_t));
```

Comment:
The documentation says that `MPI_Aint`  it is a "C type that holds any valid address." On my machine it is typedef'd as `long int`. We'd need to check that it is still 64 bits on windows.